satpy/composites/__init__.py

```diff
@@ -387,18 +387,32 @@ def __call__(self, projectables, optional_datasets=None, **info):
         """Get the reflectance part of an NIR channel. Not supposed to be used
         for wavelength outside [3, 4] µm.
         """
+        self._init_refl3x(projectables, optional_datasets)
+        proj = Dataset(self._get_reflectance(optional_datasets) * 100, **self._nir.info)
```

Comment:
If you pass also `projectables` to `_get_reflectance()` you don't need to set the `self._nir` etc. in `_init_refl3x()`

Reply:
But what about `self._nir.info`then?
Then I would need to do this again:
```
_nir, _ = projectables
```
Right?

Reply:
Yep (or `nir, _ = projectables`). Cleaner than setting `self._nir` anyways :-)